weave/legacy/weave/environment.py

```diff
@@ -251,9 +251,7 @@ def weave_wandb_api_key() -> typing.Optional[str]:
     env_api_key = _wandb_api_key_via_env()
     netrc_api_key = _wandb_api_key_via_netrc()
     if env_api_key and netrc_api_key:
-        raise errors.WeaveConfigurationError(
-            "WANDB_API_KEY should not be set in both ~/.netrc and the environment."
-        )
+        print("WANDB_API_KEY should not be set in both ~/.netrc and the environment.")
```

Comment:
* If the contents are identical I think we could be silent.
* I'd change the message to explain a bit better. Maybe something like:
"There are different credentials in the netrc file and the environment. Using the environment value." 
* Not worth fixing but just noting that NETRC file might not be located at ~/.netrc (Windows, I think we might support the NETRC env var for alternate path, etc.)

Reply:
makes sense
_wandb_api_key_via_netrc() -> i believe checks the windows location as well

Reply:
Right, just meant that on windows this message would have an incorrect path (`~/.netrc` vs. `~/_netrc`)